doc/lightningd-config.5.md

```diff
@@ -342,8 +342,9 @@ network.
 ### Networking options
 
 Note that for simple setups, the implicit *autolisten* option does the
-right thing: it will try to bind to port 9735 on IPv4 and IPv6, and will
-announce it to peers if it seems like a public address.
+right thing: for the mainnet (bitcoin) network it will try to bind to
```

Comment:
Initially thought the "bitcoin" was unnecessary but appears you do specify mainnet is Bitcoin on this project :)

Reply:
Thanks for noticing that! Yes, it is part of history here that mainnet == bitcoin, but maybe I am just too word-exact here (with alternatives). So what change would you suggest @michaelfolkson ? I will be happy to change it.

Reply:
I guess it just depends whether the implementation either today or in future could support Bitcoin-like altcoins (or Liquid maybe?). If it could it makes sense to keep referring to Bitcoin. If it doesn't just mainnet would be enough as implicitly that means Bitcoin mainnet. As I said there are lots of references to "Bitcoin mainnet" in the codebase so this isn't out of place :)

Reply:
Liquid is already supported with C-Lightning. As well as Litecoin and Litecoin-testnet. So it makes sense for that network reference to be there (and even more now because the default port mentioned is valid _only_ for Bitcoin mainnet).

Thanks for spending time thinking about it. In a way I find this issue similar to Bitcoin Core's https://github.com/bitcoin/bitcoin/pull/23306, just approached backwards, i.e. C-Lightning supported different ports early, but did not use different default port according to the selected network.